src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from 'react';
+import { useMatch } from 'react-router-dom';
+import { PeopleTable } from '../PeopleTable';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { ErrorTypes } from '../../types/ErrorTypes';
+
+export const PeoplePage = () => {
+  const [persons, setPersons] = useState<Person[]>([]);
+  const [isPersonsLoaded, setIsPersonsLoaded] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isErrorNoteShown, setErrorNoteShown] = useState(false);
+  const match = useMatch('/people/:slug');
+  const selectedPerson = match?.params.slug;
+  const isCountPersonsLoaded = persons.length === 0;
```

Comment:
the name gives wrong association. Better to name in a way that shows your intention - isPeopleListEmpty or something